vispy/ext/cassowary.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+"""
+Handle loading cassowary package from system or from the bundled copy
+"""
+
+try:
+    from ._bundled.cassowary import *  # noqa
+except ImportError:
+    import cassowary  # noqa
```

Comment:
these two import statements are not equivalent, you want `from cassowary import *` on this line